lib/lists.nix

```diff
@@ -148,7 +148,7 @@ rec {
     let found = filter pred list;
     in if found == [] then default else head found;
 
-  /* Return true iff function `pred' returns true for at least element
```

Comment:
"iff" is a short-hand for "if and only if" (using just "if" here suggests that the function could return true in other cases as well)